### DISCUSSION TITLE
The display is not normal after closing the preview

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [ibhagwan/fzf-lua#1338] [Q&A] [answered]
Open `files`, press `F4` to close the preview, and the effect will be as follows.

The same is true for other `fzf-lua` related operations.

![image](https://github.com/user-attachments/assets/cb66786f-762f-4d9d-ba7d-b3fda3bac698)

It used to be normal, and I'm not sure what operation caused the problem.

**Comment by iamxiaojianzheng** [comment]
My environment is Windows11 + Windows Terminal + Neovim-0.10 + LazyVim

**Comment by iamxiaojianzheng** [comment, marked as answer]
I accidentally found the reason for this problem, because of the fzf itself.

This happens when fzf is upgraded to `0.54.0`. When I went down to `0.53.0`, It went back to normal.

**Comment by iamxiaojianzheng** [comment]
@ibhagwan Hello, I wonder if this information can help you solve this problem?

**Reply by ibhagwan** [reply]
> This happens when fzf is upgraded to 0.54.0. When I went down to 0.53.0, It went back to normal.
>  I wonder if this information can help you solve this problem?

There isn’t much I can do about this from fzf-lua’s side, you’d need to open an issue with fzf’s repo upstream.

**Reply by iamxiaojianzheng** [reply]
This issue no longer occurs after upgrading to 0.54.2
